### PR TITLE
Add a test that fails before PR 4482

### PR DESCRIPTION
**Pull request by pirapira**
This PR adds a test on ecpairing where g1 is invalid while g2 is zero.

cpp-ethereum before PR 4482 accidentially returned some answer in this case.  It should throw.

### CODE REVIEW

File: src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalidFiller.json [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+                    "90f0b1ebbba1c1936aff7aaf20a7878ff9e04b6c": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    }
+                }
+            }
+        ],
+        "explanation": "Puts the given data into the ECPAIRING precompile"
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalidFiller.json]: we agreed to use //for comments in the fillers
and if you want the comment to end up in generated test fill the description into
```
"_info" : {
            "comment" : ""
}
```

_info.comment should be copied to the final test when filling

Comment (pirapira) [reply src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalidFiller.json]: OK.

**Comment by pirapira** [comment]
@winsvega I added a comment in `_info`.

File: GeneralStateTests/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalid.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
+{
+    "ecpairing_one_point_with_g2_zero_and_g1_invalid" : {
+        "_info" : {
+            "comment" : "",
```
Comment (winsvega) [inline GeneralStateTests/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalid.json:4]: this field should be filled by testeth if there is one in a filler file

Comment (pirapira) [reply GeneralStateTests/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalid.json:4]: testeth does not do that.  So testeth has a bug.

Comment (pirapira) [reply GeneralStateTests/stZeroKnowledge/ecpairing_one_point_with_g2_zero_and_g1_invalid.json:4]: https://github.com/ethereum/cpp-ethereum/issues/4454

**Pull request merged**